CAP-1509: Kube setup.

danieljy opened the pull request:
Refactor a bit, see readme contents.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8)
<!-- Reviewable:end -->

m12i commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 14 of 15 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---

*[coord/README.md, line 25 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7YKRPAK1Y0Cr6du0r:-LB7YKRPAK1Y0Cr6du0s:b184utk) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/README.md#L25)):*
> ```
>  you must 
> ```

you must set ?

---

*[coord/deploy/deployment-template.yaml, line 10 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7Z7-D0K_tt0OR7OJO:-LB7Z7-D0K_tt0OR7OJP:b-qb6cd6) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/deploy/deployment-template.yaml#L10)):*
> ```
> flow
> ```

`flow` or `coord` ?

---

*[coord/deploy/deployment-template.yaml, line 13 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7ZG-w-XS-oGGLoPIw:-LB7ZG-w-XS-oGGLoPIx:b-cbcdhe) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/deploy/deployment-template.yaml#L13)):*
> ```
> sandbox
> ```

sandbox ?

---

*[coord/deploy/deployment-template.yaml, line 69 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7Zb3WCuu5IGqnVXkX:-LB7Zb3WCuu5IGqnVXkY:b-d0wxhj) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/deploy/deployment-template.yaml#L69)):*
> ```YAML
>         resources:
>           requests:
>             cpu: 500m
> ```

new line

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-:-LB7bfYRCkWPFoFCKCy9:bnfp4nl)*
<!-- Sent from Reviewable.io -->

danieljy commented:
Review status: all files reviewed at latest revision, 4 unresolved discussions, all commit checks successful.

---

*[coord/README.md, line 25 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7YKRPAK1Y0Cr6du0r:-LB7cAphEG9GWhPtbP1X:b-896fix) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/README.md#L25)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

> ```
>  you must 
> ```

you must set ?
</blockquote></details>

Done.

---

*[coord/deploy/deployment-template.yaml, line 10 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7Z7-D0K_tt0OR7OJO:-LB7cHhD4q1snLpO6otV:b-xs4akt) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/deploy/deployment-template.yaml#L10)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

> ```
> flow
> ```

`flow` or `coord` ?
</blockquote></details>

Removed - this was copied/pasted.

---

*[coord/deploy/deployment-template.yaml, line 13 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7ZG-w-XS-oGGLoPIw:-LB7cL9H-cTVygIop84M:ba3dfob) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/deploy/deployment-template.yaml#L13)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

> ```
> sandbox
> ```

sandbox ?
</blockquote></details>

Removed

---

*[coord/deploy/deployment-template.yaml, line 69 at r1](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8#-LB7Zb3WCuu5IGqnVXkX:-LB7cNu18MXZub2XAqIt:b-896fix) ([raw file](https://github.com/sidewalklabs/opentripplanner/blob/e592d317bf0dcf1d0d398000010b597df50c99c1/coord/deploy/deployment-template.yaml#L69)):*
<details><summary><i>Previously, m12i (Mahmood Rahmani) wrote…</i></summary><blockquote>

new line
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/opentripplanner/8)*
<!-- Sent from Reviewable.io -->
